new_files/vendor-no-composer/robinthehood/ModifiedStdModule/Classes/StdModule.php

```diff
@@ -268,11 +268,38 @@ public function addConfiguration($key, $value, $groupId, $sortOrder, $setFunctio
         xtc_db_query("INSERT INTO `" . TABLE_CONFIGURATION . "` (`configuration_key`, `configuration_value`, `configuration_group_id`, `sort_order`, `set_function`, `use_function`, `date_added`) VALUES ('$key', '$value', '$groupId', '$sortOrder', '$setFunction', '$useFunction', NOW())");
     }
 
-    public function deleteConfiguration($key)
+    public function removeConfiguration(string $key): bool
     {
-        $key = $this->getModulePrefix() . '_' . $key;
+        $key              = $this->getModulePrefix() . '_' . $key;
+        $remove_key_query = xtc_db_query(
+            sprintf(
+                /** TRANSLATORS: %1$s: Database table "configuration". %2$s: Value for "configuration_key". */
+                'DELETE FROM `%1$s` WHERE `configuration_key` = "%2$s"',
+                TABLE_CONFIGURATION,
+                $key
+            )
+        );
+
+        $remove_backup_query = xtc_db_query(
+            sprintf(
+                /** TRANSLATORS: %1$s: Database table "configuration". %2$s: Value for "configuration_key". */
+                'DELETE FROM `%1$s` WHERE `configuration_key` = "%2$s"',
+                defined('TABLE_MODULE_BACKUP') ? TABLE_MODULE_BACKUP : 'module_backup',
+                $key
+            )
+        );
```

Comment:
We can't do this, because most users do

- backup
- uninstall
- install
- import backup

With this new code:

- backup
- uninstall
- remove backup ⚠️
- install
- import backup ☠️

Reply:
Hehehehe thanks. I've removed it entirely now since it would be out-of-scope for this PR. But I like the idea of an extra method.